### DISCUSSION TITLE
vs code

### DISCUSSION THREAD

**Discussion by Yanntak2** [community/community#67673] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

comment remettre les paramètres de base d'usine dans vs code

**Comment by seyLu** [comment]
#### Hi  @Yanntak2,

If you mean just reset settings to default, then on your `settings.json`, replace the contents with two curly braces `{}`.

If you also want to reset keybindings in VSCode, then on your `keybindings.json`, replace the contents with two curly braces `{}`. 

If you also want to delete all extensions, then just delete extensions dir.

You can read more on:
- [VSCode docs: Reset User Settings](https://code.visualstudio.com/docs/getstarted/settings#_how-can-i-reset-my-user-settings)
- [Reset all Keyboard Shortcuts in VSCode](https://bobbyhadz.com/blog/reset-keyboard-shortcuts-in-vscode#reset-all-keyboard-shortcuts-in-vs-code)
- [VSCode docs: Where are extensions located](https://code.visualstudio.com/docs/editor/extension-marketplace#_where-are-extensions-installed)
